INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
```

Comment:
It looks more professional if we have correct capitalization, especially for our own name. :slightly_smiling_face: